velocity/tool/src/java/org/sakaiproject/cheftool/menu/MenuEntry.java

```diff
@@ -306,6 +309,15 @@ public boolean getIsField()
 
 	} // getIsField
 
+	public boolean getIsCurrent()
+	{
+		return m_current;
+	}
+
+	public void setIsCurrent(boolean current) {
+		m_current = current;
+	}
+
 	@Override
 	public String getAccessibilityLabel() {
 		return ((m_accessibilitylabel == null) ? m_title : m_accessibilitylabel);
```

Comment:
Yes I meant the parentheses outside of the ternary , please sorry for my late reply.